sdk/ai/azure-ai-resources/tests/connections/unittests/test_connection_entity.py

```diff
@@ -0,0 +1,43 @@
+import random
+import string
+
+import openai
+import pytest
+
+from azure.ai.resources.entities import AzureOpenAIConnection, AzureAISearchConnection, AzureAIServiceConnection
+from azure.ai.ml._restclient.v2023_06_01_preview.models import ConnectionAuthType
+from azure.ai.ml._utils.utils import camel_to_snake
+from azure.ai.ml.entities._credentials import ApiKeyConfiguration
+from azure.core.exceptions import ResourceNotFoundError
+
+
+@pytest.mark.e2etest
+class TestConnections:
+
+
+
+    def test_aoai_entity(self):
+        pass
+
+
+    def test_search_entity(self):
+        pass
+
+    
+    def test_service_entity(self):
+        pass
+ 
+    # TODO move to unit tests
+    def test_set_keys(self):
```

Comment:
Test makes no assertions. Unless the goal is that it doesn't throw an exception?